Apps/phpassivetotal/passivetotal.json

```diff
@@ -1819,6 +1903,1015 @@
+                    "example_values": [
+                        "2018-09-01"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.ssl_certificates.*.firstSeen",
+                    "data_type": "string",
+                    "example_values": [
+                        "2018-08-01"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.ssl_certificates.*.ipAddresses",
+                    "data_type": "string",
+                    "example_values": [
+                        "8.8.8.8"
+                    ]
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "City: San Francisco, Country: us, Organization: Abc, Inc."
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string",
+                    "example_values": [
+                        "Total records: 10"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "width": 12,
+                "title": "Lookup Certificate Hash",
+                "type": "table",
+                "height": 5
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "lookup certificate",
+            "description": "Lookup certificate",
+            "type": "investigate",
+            "identifier": "lookup_certificate",
+            "read_only": true,
+            "parameters": {
+                "query": {
+                    "description": "Field value for which to search certificates",
+                    "data_type": "string",
+                    "order": 0,
+                    "required": true
+                },
+                "field": {
+                    "description": "The field for which to search certificates",
+                    "data_type": "string",
+                    "order": 1,
+                    "required": true,
+                    "value_list": [
```

Comment:
Do you have to search in a field or are you able to search across all fields?

Reply:
It takes inputs as a field from the list and its value and finds certificates that contain the value we have passed for the field